src/stryker.ts

```diff
@@ -1,13 +1,23 @@
 import * as vscode from 'vscode';
 import { strykerCommand, strykerConfigFilePath } from './config';
 import { makeReusableTerminal, runCommand } from './terminal';
+import path from 'path';
 
 export type CommandRunner = (args: { file: vscode.Uri; lineRange?: string }) => void;
 
+const getRelativePath = (file: vscode.Uri): string => {
+  const workspaceFolder = vscode.workspace.getWorkspaceFolder(file);
+  if (workspaceFolder) {
+    return path.relative(workspaceFolder.uri.fsPath, file.fsPath);
+  } else {
```

Comment:
Suggestion: we could lose this `else` block entirely as there is an early return in the initial `if` block.

```
if (workspaceFolder) {
    return path.relative(workspaceFolder.uri.fsPath, file.fsPath);
  }
return file.fsPath;
```

Reply:
I applied your suggestion ☺